app/assets/javascripts/ga_cross_domain_form_tracking.js

```diff
@@ -0,0 +1,17 @@
+function attachCrossDomainTrackerToInput(form, trackers) {
+  if (trackers.length) {
+    var existingAction = form.attr("action");
+    var linker = new window.gaplugins.Linker(trackers[0])
+    var trackedAction = linker.decorate(existingAction)
+    form.attr("action", trackedAction);
+  }
+}
+
+$(window).on('load', function() {
+  if (ga !== undefined && typeof ga.getAll === 'function') {
+    var form = $("form#account-signup");
```

Comment:
We should probably check that `form` exists before proceeding here. I think `if (form.length)` should do it?